firmware/projects/DemoProject/platforms/stm32f767/cubemx/board_config.ioc

```diff
@@ -21,8 +21,8 @@ Mcu.PinsNb=3
 Mcu.ThirdPartyNb=0
 Mcu.UserConstants=
 Mcu.UserName=STM32F767ZITx
-MxCube.Version=6.7.0
-MxDb.Version=DB.6.0.70
+MxCube.Version=6.10.0
```

Comment:
What version of cubemx are we going to use? I specified 6.8.1 in the firmware readme and we all need to be synchronized.